templates/snap-details.html

```diff
@@ -101,6 +115,47 @@ <h4>Where people are using {{ snap_title }}</h4>
       whitelistUrls: ['snapcraft.io']
     }).install();
     Raven.context(function () {
+      if (typeof ga !== 'undefined') {
```

Comment:
GA is loaded asynchronously so in many cases this will run before QA is loaded and will silently fail.

We should only check for GA at the moment we attempt to use it and initialise everything else normally to have event handlers in place regardless of GA being defined.